.github/workflows/test.yml

```diff
@@ -55,3 +91,7 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           file: ./coverage.xml
+
+      - name: Stop docker-compose services
```

Comment:
Do we really need to stop them? Or can we just end the build?

Reply:
I am not 100% sure, so I will double-check it, but personally I think it's good practice to clean resources explicitly